src/HTMLGenerator/testSuite/sequencer.py

```diff
@@ -3,18 +3,76 @@
 import pathlib
 import errno
 import json
+import time
 from pathlib import Path
 from asserter import TestCase
 from tqdm import tqdm
 import utils
+import subprocess
 from utils import *
 import sys
 sys.path.append("..")
-import converter
-from converter import convert
 
+CURRENT_CONFIGURATION_PATH = "currentConfiguration.txt"
+NOTIFICATION_PATH = "complete.txt"
+LABVIEW_PATH = Path("C:/Program Files/National Instruments/LabVIEW 2017/LabVIEW.exe")
 ERROR_REPORT_JSON = r'errorReport.json'
 
+def _get_path_relative_to_username(filename):
+    # Return a path that is relative to the ~user
+    s = Path("~") 
+    filename_path = Path(filename)
+    return str(filename_path.relative_to(s.expanduser()))
```

Comment:
If this is the exact same thing present in converter.py, I would recommend moving it to utils.py.

Reply:
Okay

Reply:
No longer needed, closing this comment.